static/js/containers/ProfilePage_test.js

```diff
@@ -1,4 +1,6 @@
 /* global SETTINGS: false */
+import '../global_init';
```

Comment:
I swear there was a way I was running the tests where it failed without that import, but I can't reproduce it or remember exactly what  the error was. I'll remove it.